packages/coinstac-images/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Assuming all these subpackages are built as Docker images, so they won't have a _node_modules_ dir, causing the eslint-plugin-import to error. Disabling the rule seems like a sensible action.

Reply:
Yeah... I've been on the fence of how they should work, I think they're just fine as a free floating dir in packages that are then build as images. Maybe some update scripts to keep things consistent and make pubing images easier